pykechain/models/property.py

```diff
@@ -366,6 +366,21 @@ def validate(self, reason: bool = True) -> List[Union[bool, Tuple]]:
         else:
             return self._validation_results
 
+    def has_validator(self, validator_klass: Type[PropertyValidator]) -> Optional[bool]:
+        """
+        Check if any validator matches the given class.
+
+        :param validator_klass: The class of the validator to check.
+        :type validator_klass: Type[PropertyValidator]
+        :return: True if a matching validator exists, False otherwise, or None if no validators are set.
+        :rtype: bool or None
+        """
+        if not self._validators:
+            return None
+        return any(
+            isinstance(validator, validator_klass) for validator in self._validators
+        )
```

Comment:
Why this function's return type is `Optional[bool]`? 

It could be just `bool`.